README.md

```diff
@@ -0,0 +1,5 @@
+# Flame tutorials
+
+This repository includes a collection of official tutorials created and maitained by the Flame team. Use the index below to navigate throgh the availables tutorials.
+
+ - [Basic: Rendering a simple square on the screen](./tutorials/basic_1_square/README.md)
```

Comment:
maybe we should do numbered tutorials? the expectation is that people follow in order right? like chapters